### PR TITLE
fix: renamed autocomplete prop to camelcase

### PR DESCRIPTION
**Pull request by s-barrah**
fix: renamed autocomplete prop to camelcase

### CODE REVIEW

File: src/components/InputField/InputField.js [hunk 1]
```diff
@@ -276,7 +276,7 @@ InputField.defaultProps = {
   value: null,
   pattern: '',
   required: false,
-  autocomplete: '',
+  autoComplete: '',
```
Comment (Heleen-cr) [inline src/components/InputField/InputField.js]: Does the empty string has the same result as not having the autocomplete attribute to the element?

I thought it's not necessary to have a default value if the property is not required.
See how other properties are used for the attributes in the input field above.

File: src/components/InputField/InputField.js [hunk 2]
```diff
@@ -220,7 +220,7 @@ class InputField extends Component {
               name={this.props.name && this.props.name}
               className={`form__field form__field--${this.props.type} ${error}`}
               required={this.props.required}
-              autocomplete={this.props.autocomplete}
+              autoComplete={this.props.autoComplete}
```
Comment (Heleen-cr) [inline src/components/InputField/InputField.js]: The actual html attribute isn't camel-cased though.

Comment (s-barrah) [reply src/components/InputField/InputField.js]: But React prefers it to be camelCase and works the same. Same thing applies for a few html properties. I have tested this.

File: src/components/InputField/InputField.js (hunk 1 above)
Comment (s-barrah) [reply src/components/InputField/InputField.js]: It does no harm to have as a default prop even if it is not required.

File: src/components/InputField/InputField.js (hunk 2 above)
Comment (Heleen-cr) [reply src/components/InputField/InputField.js]: ok cool didn't know react prefers it

Comment (s-barrah) [reply src/components/InputField/InputField.js]: Screen Shot 2019-03-13 at 12 43 48

File: src/components/InputField/InputField.js (hunk 1 above)
Comment (Heleen-cr) [reply src/components/InputField/InputField.js]: that's true, was just wondering whether the empty string might have an effect, but tested and it seems to be fine.

**Pull request merged**